### DISCUSSION TITLE
cvxpy and moske issue

### DISCUSSION THREAD

**Discussion by NUSashwinmk** [cvxpy/cvxpy#1480] [General]
I have installed mosek using conda install. however when I use
import cvxpy, I get an error 
AttributeError: module 'mosek' has no attribute 'Env'

looks like there is problem when mosek is being imported

~\Anaconda3\lib\site-packages\pulp\apis\__init__.py in <module>
      5 from .choco_api import *
      6 from .mipcl_api import *
----> 7 from .mosek_api import *

How do I get rid of mosek from cvxpy?

**Comment by SteveDiamond** [comment]
This error is common when using an old version of mosek. You could uninstall mosek. But note that the error you cite is coming from pulp, not cvxpy.
